src/dyn_subgrid/dyncropFileMod.F90

```diff
@@ -174,6 +175,12 @@ subroutine dyncrop_interp(bounds,crop_inst)
     ! The call collapse_crop_types also appears in subroutine surfrd_veg_all
     call collapse_crop_types(wtcft_cur, fertcft_cur, cft_size, bounds%begg, bounds%endg, verbose = .false.)
 
```

Comment:
I kept this new subroutine, even though now it's only tangentially relevant because @billsacks said it would be helpful for crop variables. We can discuss whether to keep.